File: src/SOP8/Base/Account/Base.Account.IBLL/Response/ResponseAccountUser.cs
using Response;

namespace Base.Account.IBLL.Response
{
    using Models;

    public class ResponseAccountUser : MessageResult
    {
        private AccountUser m_user = null;

        public AccountUser User
        {
            get { return m_user; }
            set { m_user = value; }
        }

        public ResponseAccountUser()
            : base()
        {
        }

        public ResponseAccountUser(bool success, string message)
            : base(success, message)
        {
        }
    }
}

File: src/SOP8/Egovframework/Base/base-account-ibll/src/main/java/egovframework/base/account/ibll/response/ResponseAccountUser.java
package egovframework.base.account.ibll.response;

import egovframework.base.account.ibll.models.AccountUser;
import egovframework.base.response.MessageResult;

public class ResponseAccountUser extends MessageResult {

    private AccountUser user = null;

    public AccountUser getUser() {
        return user;
    }

    public void setUser(AccountUser user) {
        this.user = user;
    }

    public ResponseAccountUser() {
        super();
    }

    public ResponseAccountUser(boolean success, String message) {
        super(success, message);
    }
}